backend/battling/views.py

```diff
@@ -115,11 +115,17 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         battle = self.get_object()
 
-        creator = Team.objects.get(battle=battle, trainer=battle.creator.id)
-        context["creator_team"] = creator.pokemons.all()
-
-        opponent = Team.objects.get(battle=battle, trainer=battle.opponent.id)
-        context["opponent_team"] = opponent.pokemons.all()
+        creator_pokemon_list = PokemonTeam.objects.filter(
+            team__battle=battle, team__trainer=battle.creator
+        ).select_related("pokemon")
+        creator_pokemon_list.order_by("order")
```

Comment:
This is wrong because queryset methods never mutate the object in the left side (the object you're applying the `.` into).
The correct line here is: `creator_pokemon_list = creator_pokemon_list.order_by("order")`

Also, I would change the name `creator_pokemon_list` to `creator_pokemon_qs`, because this is not a list, but a queryset.

Right now this is only working for you due to chance.